autoblocks/_impl/api/client.py

```diff
@@ -230,3 +233,44 @@ def get_dataset(
             revision_id=resp["revisionId"],
             items=[DatasetItem(id=item["id"], splits=item["splits"], data=item["data"]) for item in resp["items"]],
         )
+
+    def get_local_test_runs(self, test_external_id: str) -> List[AutoblocksTestRun]:
+        req = self._client.get(f"/testing/local/tests/{encode_uri_component(test_external_id)}/runs")
+        req.raise_for_status()
+        resp = req.json()
+        return [AutoblocksTestRun(id=run["id"]) for run in resp["runs"]]
+
+    def get_ci_test_runs(self, test_external_id: str) -> List[AutoblocksTestRun]:
+        req = self._client.get(f"/testing/ci/tests/{encode_uri_component(test_external_id)}/runs")
+        req.raise_for_status()
+        resp = req.json()
+        return [AutoblocksTestRun(id=run["id"]) for run in resp["runs"]]
+
+    def get_local_test_run_results(self, run_id: str) -> List[AutoblocksTestCaseResult]:
+        resp = self._client.get(f"/testing/local/runs/{encode_uri_component(run_id)}/results")
+        resp.raise_for_status()
+        data = resp.json()
+        return [AutoblocksTestCaseResult(
+                id=result["id"]
+            )
+            for result in data["results"]
+        ]
+
+    def get_ci_test_run_results(self, run_id: str) -> List[AutoblocksTestCaseResult]:
+        resp = self._client.get(f"/testing/ci/runs/{encode_uri_component(run_id)}/results")
+        data = resp.json()  # Parse the response JSON
+        return [AutoblocksTestCaseResult(
+            id=result["id"]
+        ) for result in data["results"]]
+
+    def get_local_test_case_result(self, test_case_result_id: str) -> AutoblocksTestCaseResultWithEvaluations:
```

Comment:
let's make sure we are consisenst between the js and python sdk so here we have `get_local_test_case_result` but in the js sdk we call it test result.

I think on both we should do test case result.